.github/workflows/publish.yml

```
@@ -0,0 +1,114 @@
+
+      - run: yarn
+
+      - name: build
+        run: yarn build
+
+      #Add remove-stable-version
+      - name: Bump query-subgraph & deploy
+        uses: ./.github/actions/create-prerelease
+        with:
+          package-path: .
+          npm-token: ${{ secrets.NPM_TOKEN }}
```

actionlint (1.7.9)

input "npm-token" is not defined in action "Remove Stable Versions" defined at "./.github/actions/create-prerelease". available inputs are "package-path"

(action)